prelims/Program_Demo.R

```diff
@@ -1,6 +1,6 @@
 # Initialize a list where student data will be contained within vectors.
 # Vectors shall contain the following: ID Num, Name, and Status
-student_list <- list(c(2223175, "Rohit Tank", "Present”));
```

Comment:
Please refer to Section II of the company's coding standards under Syntax - Semi colons, and ensure compliance with it. Make sure to use the updated curly brace format consistently across the entire code to meet the modified standard. Thank you.